### ISSUE TITLE
Suggestion: multiline for default values

### ISSUE THREAD

**Issue by tanguilp** [Qqwy/elixir_confy#2]
It'd be nice if "big" default values (or default values with complex data structures) would be displayed in a multiline highlighted code block instead of inline, as shown in the this screenshot:

![Screenshot from 2020-06-05 18-07-50](https://user-images.githubusercontent.com/29804907/83892761-c78b2600-a757-11ea-9932-8ff398eb8d7f.png)

The question is how to determine which default values are to be displayed inline and which are to be displayed in a code block. The example above uses the following heuristics: code block is used when byte size of the string representation of the code is > 15 bytes.

What do you think?

**Comment by Qqwy** [comment]
Great idea!

We can use a slightly more advanced heuristic, because Elixir already has the pretty-printing built-in, which is what we will use to fill the code block to begin with. So we could do something like:

```elixir
n_lines = 
  value
  |> Kernel.inspect(printable_limit: :infinity, limit: :infinity, width: 80, pretty: true)
  |> String.split("\n")
  |> Enum.count()

if n_lines > 1 do
  # single line
else
  # multi line
end
```

Would that fit your purposes?

**Comment by tanguilp** [comment]
Absolutely, great idea!

**Comment by tanguilp** [comment]
Or even using `|> String.contains("\n")`, might be more readable.

**Comment by Qqwy** [comment]
This change has been published in version 0.7.1!

Thank you very much for your suggestion :green_heart:~
